.pre-commit-config.yaml

```diff
@@ -22,8 +22,6 @@ repos:
           examples/.*|
           \.gitignore|
           CONTRIBUTING\.md|
-          DOCS\.md|
           LICENSE\.txt|
-          PRECOMMITHOOKS\.md|
```

Comment:
Why would you like to remove `PRECOMMITHOOKS.md`?

Reply:
This PR also moves the `PRECOMMITHOOKS.md` to the `docs/` directory. And everything within `docs/` is already ignored. Thus I removed the redundant reference.

https://github.com/lightly-ai/lightly/blob/745fed3ef146380e7f515ec7c2c513ffbd6014fc/.pre-commit-config.yaml#L21

Reply:
I think we should keep `PRECOMMITHOOKS.md` outside of the `docs` directory as `docs` is used for our documentation hosted here: https://docs.lightly.ai/self-supervised-learning/ and not for internal documentation.

Reply:
@SauravMaheshkar could you add `PRECOMMITHOOKS.md` back here?